src/content/contributing/adding-desci-projects/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Adding DeSci projects
+description: The policy we use when adding a links to projects on the DeAci page on ethereum.org
+lang: en
+sidebar: true
+---
+
+# Adding projects {#adding-projects}
+
+We want to make sure we show a variety of projects and give a good snapshot of the DeSci landscape.
+
+Anyone is free to suggest a project to list on the DeSci page on ethereum.org. Equally, anyone who notices a project that is no longer relevant or no longer meets our eligibility criteria, they are free to suggest we remove it.
+
+## The decision framework {#the-decision-framework}
+
+### Criteria for inclusion: the must-haves {#the-must-haves}
+
+- **Open source code/data** - Openness of code and data is a core DeSci principle, so DeSci projects must not be closed source. The codebase should be accessible and ideally open to PRs.
+- **DeSci projects should be demonstrably decentralized** - This include being governed by a DAO, or by building with decentralized tech stack including non-custodial wallets. It probably involves auditable smart contracts at publicly-known Ethereum addresses.
+- **Honest and accurate listing information** - it is expected that any suggested listings from projects come with honest and accurate information. Products that falsify listing information, such as declaring your product is “open source” when it is not, will be removed.
+- **Demonstrable commitment to widening access to science** - A DeSci project should be able to articulate how they widen participation in science to the general public, not just to token/NFT holders.
+- **Point of contact** - A point of contact for the project (this might be by a representative from a DAO or community) will greatly help us get accurate information when changes are made. This will keep updating ethereum.org manageable when gathering future information.
+- **Globally accessible** - your project doesn’t have geographic limitations or KYC requirements that exclude certain people from accessing your service.
+- **Informative website and documentation** - it is important that visitors to the project website can understand what the project actually does, how it contributes to decentralization of science infrastructure and how to participate.
+
+### Nice-to-haves
+
+- **Available in multiple languages** - your project is translated into multiple languages allowing users around the world to access it.
+- **Educational resources** - your product should have a well-designed onboarding experience to help and educate users. Or evidence of how-to content like articles or videos.
+- **Third party audits** - your product has been professionally audited for vulnerabilities by a trusted third party.
```

Comment:
```suggestion
- **Third party audits** - your product has been professionally audited for vulnerabilities by a trusted third party.
- **Point of contact** - A point of contact for the project (this might be by a representative from a DAO or community) will greatly help us get accurate information when changes are made. This will keep updating ethereum.org manageable when gathering future information.
```